plugins/destination/duckdb/client/write.go

```diff
@@ -76,9 +76,9 @@ func (c *Client) upsert(ctx context.Context, tmpTableName string, table *schema.
 		if written > 0 {
 			sb.WriteString(", ")
 		}
-		sb.WriteString(col.Name)
+		sb.WriteString(sanitizeID(col.Name))
```

Comment:
`binary` is now(? updated duckdb driver probably?) failing here so we need to sanitize all.